dev-libs/log4cxx/log4cxx-1.2.0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+MY_P=apache-${P}
+DESCRIPTION="Library of C++ classes for logging to files, syslog and other destinations"
+HOMEPAGE="https://logging.apache.org/log4cxx/"
+SRC_URI="mirror://apache/logging/${PN}/${PV}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="Apache-2.0"
+SLOT="0/15"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~x86"
+IUSE="odbc smtp test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/apr:1=
+	dev-libs/apr-util:1=
+	dev-libs/libfmt:=
+	odbc? ( dev-db/unixODBC )
+	smtp? ( net-libs/libesmtp )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	app-arch/zip
+	test? (
+		app-alternatives/gzip
+		app-arch/zip
+	)
+"
+
+src_prepare() {
+	# https://github.com/apache/logging-log4cxx/issues/189
```

Comment:
There are proper build system options with this in the new release. Please always carefully review `pkgdiff-mg -b` output and adapt the ebuild accordingly.

Reply:
I believe I've fixed this in the latest commit, thank you for telling me about `pkgdiff-mg` though, made things way easier!